validator/webui/index.html

```diff
@@ -299,18 +301,35 @@
         background-size: 40px 40px;
         background-position: center left;
       }
-      .ampproject a:link { color: #0379C4; text-decoration: none; }
-      .ampproject a:visited { color: #0379C4; text-decoration: none; }
-      .ampproject a:hover { color: #0379C4; text-decoration: none; }
-      .ampproject a:active { color: #0379C4; text-decoration: none; }
+      .ampproject-right {
+        color: #0379C4;
+        margin: 10px 0;
+      }
+      .ampproject-link a:link { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:visited { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:hover { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:active { color: #0379C4; text-decoration: none; }
```

Comment:
Ah, yes, there's a way around that but it's really just a nit at this point. LGTM